src/types/options.ts

```diff
@@ -0,0 +1,276 @@
+// Minimum TypeScript Version: 3.8
+
+import { OktaAuth, OktaAuthOptions } from '@okta/okta-auth-js';
+import {
+  SimpleCallback,
+  RenderResult,
+  RenderError,
+} from './results';
+import { RegistrationOptions } from './registration';
+
+export interface WidgetOptions extends Partial<Pick<OktaAuthOptions,
+  'issuer' | 
+  'clientId' | 
+  'redirectUri' |
+  'state' | 
+  'scopes' |
+  'codeChallenge' |
+  'codeChallengeMethod' |
+  'flow'
+>> {
```

Comment:
merge conflict?

Reply:
Nope, it's just a generic inside a generic.  `Partial<Pick<X>>`